### PR TITLE
DOCS-299 Create a "Community SDKs" Page

### PR DESCRIPTION
**Pull request by alexisintech**
To honor Clerk users and contributors, and to celebrate our Clerk community, we want to highlight community-maintained Clerk integrations.

This PR

- adds a Community section to our homepage nav
- adds a Community SDK's page
- fixes some typos/inconsistencies with text

See addition of "Community" section on left side nav (main page). Note that when clicking on "Community SDK's", the path is /docs/references/community and the context does not change (the left side nav remains the same).

Screenshot 2023-09-05 at 12 17 59

---

Waiting on 

- [x] Icons for Elysia, Rust, and Vue are added
- [x] "Community" badge is created
- [ ] Community SDK's get their own links under "SDK References"
- [ ] Remove the "Community" section from the nav

### CODE REVIEW

File: docs/manifest.json [hunk 1]
```diff
@@ -749,5 +749,9 @@
       ]
     ]
   ],
-  ["Backend API", "https://clerk.com/docs/reference/backend-api"]
+  ["Backend API", "https://clerk.com/docs/reference/backend-api"],
+  "# Community",
+  ["Community SDK's", "references/community"],
```
Comment (BRKalow) [inline docs/manifest.json:754]: Does this need to have an apostrophe? I think no, but I'm not positive...

File: docs/references/community.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: Community SDK's
```
Comment (BRKalow) [inline docs/references/community.mdx:2]:
```suggestion
title: Community SDKs
```

File: docs/references/community.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+description: Explore SDK's created by the Clerk community.
+---
+
+# Community SDK's
```
Comment (BRKalow) [inline docs/references/community.mdx:6]:
```suggestion
# Community SDKs
```

File: docs/references/community.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Explore SDK's created by the Clerk community.
```
Comment (BRKalow) [inline docs/references/community.mdx:8]:
```suggestion
Explore SDKs created by the Clerk community.
```

**Comment by panteliselef** [comment]
@alexisintech looks nice, I think u can remove the margin from the "cards". So that the text aligns properly with the grid below. At least this is how we have it for the "Welcome to Clerk Docs" page
image

**Comment by alexisintech** [comment]
> @alexisintech looks nice, I think u can remove the margin from the "cards". So that the text aligns properly with the grid below. At least this is how we have it for the "Welcome to Clerk Docs" page image

this is already a ticket in the works! :)

**Comment by austincalvelage** [comment]
Just added the UI updates for these cards in the marketing repo - https://github.com/clerkinc/clerk-marketing/pull/766

image

image


Make sure that PR gets merged before this one!

**Comment by austincalvelage** [comment]
Just added the UI updates for navigation links and badges here : https://github.com/clerkinc/clerk-marketing/pull/768

**Review by austincalvelage — APPROVED** [review]

**Pull request merged**